python-test/features/integration.feature

```diff
@@ -147,7 +147,7 @@ Scenario: Sink with invalid endpoint
         And referred agent is subscribed to a group
         And that a sink with invalid endpoint already exists
         And that a policy using: handler=dns, description='policy_dns', host_specification=10.0.1.0/24,10.0.2.1/32,2001:db8::/64, bpf_filter_expression=udp port 53, pcap_source=libpcap, only_qname_suffix=[.foo.com/ .example.com], only_rcode=2 already exists
-    When a new dataset is created using referred group, sink and policy ID
+    When a new dataset is created using referred group, policy and 1 sink ID
```

Comment:
It should not be "... group, policy ID and 1 sink"? 
or the way it was" ... group, 1 sink and policy ID"